spec/features/user_creates_voter_spec.rb

```diff
@@ -24,7 +24,7 @@
       fill_in "State", with: "MA"
       fill_in "Zip", with: "01234"
 
-      click_button 'Submit'
+      click_button 'Add this voter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.